### PR TITLE
[Nice to have in 2.4] Fix/2092 logo route

### PR DESCRIPTION
**Pull request by metzgegu**
#2092 

## Checkpoints

The goal of this section is to help code reviewer to do required checks on this pull request. Please don't edit this list.
If one or more of these checkpoints are out of scope, please write below why they are.

- [x] Code is clear enough
- [x] If there are FIXME in the code, then there are associated issues and issue is referenced in the FIXME
- [x] If there are TODO, NOTE or HACK in code, date and developer initials are present
- [x] Automated tests have been written (covering feature or non regression), or *at least* an issue has been created for test implementation

### CODE REVIEW

File: frontend/src/component/Header/Logo.jsx [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
-    </a>
+    </Link>
   )
 }
 export default Logo
 
 Logo.propTypes = {
   logoSrc: PropTypes.string.isRequired,
-  onClickImg: PropTypes.func.isRequired
+  to: PropTypes.string
```
Comment (Skylsmoi) [inline frontend/src/component/Header/Logo.jsx]: for each non required props, you should have a defaultProps value.
Also check that the default props works, meanings clicking on the logo without having a `to` specified to `<Logo />` doesn't generate a js error

**Comment by metzgegu** [comment]
defaultprops values added

**Comment by metzgegu** [comment]
Can't add Cypress test because Cypress don't support middle click

**Comment by Skylsmoi** [comment]
we will do a unit test

**Pull request merged**